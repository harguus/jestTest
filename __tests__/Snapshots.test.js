import 'react-native';
import React from 'react';
import Snapshot from '../views/Snapshots';

import renderer from 'react-test-renderer';

it('Não ha mudanças', () => {
    const snapshot = renderer.create(
        <Snapshot />
    ).toJSON();
    // cria um snap e testa mudanças
    expect(snapshot).toMatchSnapshot();
});