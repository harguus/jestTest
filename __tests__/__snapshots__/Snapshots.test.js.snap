// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Não ha mudanças 1`] = `
<View>
  <Text>
    Snapshots
  </Text>
</View>
`;
